### ISSUE TITLE
Duplicate tags if they are defined on @Operation annotation on swagger-ui

### ISSUE THREAD

**Issue by jamataran** [springdoc/springdoc-openapi#436]
**Describe the bug**
Duplicate definition when the annotations are given in the interface.

**To Reproduce**

1. Define the Interface

```
public interface AperturaCasoAPI {


    @Operation(summary = "Creación de un nuevo caso",
            tags = {"apertura"},
            requestBody = @RequestBody(description = "Datos del caso", required = true),
            responses = {
                    @ApiResponse(
                            content = @Content(
                                    mediaType = "application/json",
                                    schema = @Schema(implementation = AperturaOutDTO.class))),
                    @ApiResponse(responseCode = "400", description = "Invalid ID supplied"),
                    @ApiResponse(responseCode = "404", description = "Pet not found"),
                    @ApiResponse(responseCode = "405", description = "Datos de entrada incorrectos")}
    )
    ResponseEntity<AperturaOutDTO> apertura(AperturaInDTO aperturaIn);

}
```
2. Implement the interface

```
@RestController
@RequestMapping(value = "/apertura")
public class AperturaCasoRestController implements AperturaCasoAPI {

    @Override
    @PostMapping
    public ResponseEntity<AperturaOutDTO> apertura(AperturaInDTO aperturaIn) {
        return ResponseEntity.ok(AperturaOutDTO.builder().build());
    }
}
```

**Expected behavior**

Only one map for `apertura` endpoint. Actually, swagger-ui is showing two endPoints but only there are one.

**Screenshots**

![image](https://user-images.githubusercontent.com/14136459/74645881-0b422b80-5179-11ea-95b6-93cdcf0054a0.png)

**Additional context**
N/A

**Comment by bnasslahsen** [comment]
Hi @jamataran, 

 This is due to the fact that the tags are already defined on Operation level (And not the fact of annotations on interface).
 Anyway, this will be fixed for the next release: v1.2.31

**Comment by jamataran** [comment]
thank you  soy much !
